metrics/metric.py

```
@@ -0,0 +1,887 @@
+                url = f'https://api.semanticscholar.org/graph/v1/paper/search?query={keyword}&fieldsOfStudy=Computer Science&fields=title,year,citationCount,influentialCitationCount&offset={100 * i}&limit=100'
+
+            if url in cache:
+                r = cache[url]
+            else:
+
+                if s2api is not None:
+                    headers = {
+                        'x-api-key': s2api
+                    }
+                else:
+                    headers = None
```

Ruff (0.8.2)

Use ternary operator `headers = {'x-api-key': s2api} if s2api is not None else None` instead of `if`-`else`-block

Replace `if`-`else`-block with `headers = {'x-api-key': s2api} if s2api is not None else None`

(SIM108)